### DISCUSSION TITLE
更新3.14.0后无法发送信息

### DISCUSSION THREAD

**Discussion by GuihongWang** [koishijs/koishi#357] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57795156/130339269-657f19df-799a-4a12-86b2-48570e9b9367.png)
对应的信息：
Debian Linux 10 aarch64
node 14.17.5 yarn 1.22.5
koishi.config.js:[koishi.config.js.txt](https://github.com/koishijs/koishi/files/7026442/koishi.config.js.txt)
cqhttp的配置：[config.yml.txt](https://github.com/koishijs/koishi/files/7026443/config.yml.txt)

**Comment by shigma** [comment]
之前是哪个版本？

**Reply by GuihongWang** [reply]
之前是3.13.1

**Reply by shigma** [reply]
感觉是 bug，我建个 issue 吧

**Reply by GuihongWang** [reply]
> 感觉是 bug，我建个 issue 吧

即便是 全新安装的3. 14. 0 都存在这个bug

**Comment by GuihongWang** [comment, marked as answer]
我怎么样配置图片搜索？
终端机提示：
![image](https://user-images.githubusercontent.com/57795156/130548206-8d2a63fd-cc02-47a1-bcec-7ab68b9b8aa0.png)
koishi.config.js：
![image](https://user-images.githubusercontent.com/57795156/130548365-d7e64bf5-8e67-401b-8632-333eda538f8b.png)

**Reply by shigma** [reply]
你可能对 js 的语法不是很熟悉，对象字面量每个键值之后要打逗号：

```js
lowSim: 40,
highSim: 60,
apiKey: 'xxx', // 字符串用单引号或者双引号写即可
```

**Reply by GuihongWang** [reply]
> 你可能对 js 的语法不是很熟悉，对象字面量每个键值之后要打逗号：
> 
> ```js
> lowSim: 40,
> highSim: 60,
> apiKey: 'xxx', // 字符串用单引号或者双引号写即可
> ```

感谢 （看来要去看看js语法了）

**Comment by GuihongWang** [comment]
&Puppeteer 插件无法在 chromium no-sandbox下使用

**Comment by GuihongWang** [comment]
＆当我在使用图片搜索的时候 提示 无法连接服务器。 我怎么解决？（已经排查了是阿里云dns的锅 但修改dns后 还是一样）
